File: src/main/kotlin/br/com/fiap/techfood/adapters/outbound/repository/OrderRepository.java
package br.com.fiap.techfood.adapters.outbound.repository;

import br.com.fiap.techfood.adapters.outbound.repository.entity.OrderEntity;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface OrderRepository extends JpaRepository<OrderEntity, UUID> {
}

File: src/main/kotlin/br/com/fiap/techfood/adapters/outbound/repository/OrderRepository.kt
package br.com.fiap.techfood.adapters.outbound.repository

import br.com.fiap.techfood.adapters.outbound.repository.entities.OrderEntity
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface OrderRepository : JpaRepository<OrderEntity, UUID> {

    fun findAllByStatus(status: Int?): List<OrderEntity>

}
